stage-1-project-discovery.md

```diff
@@ -1,3 +1,66 @@
 # Stage 1: Project Discovery
 
-TODO
+> WORK IN PROGRESS: none of this should be considered final
+
+> Previous: [Stage 0: Getting Started](./stage-0-getting-started.md) | Next: [Stage 1: Project Discovery](./stage-1-project-discovery.md)
+
+In this stage you will work to gain a high level overview of what sort of
+projects there are to work on.  The goal is to avoid diving too deep on any
+single subject.
+
+Along the way, the hope is for you to start building and improving the following skills.
+
+- Feeling comfortable in the R&D Discord
+- Flex your writing skills.  Note taking and writing down what you know in an organized manner.
+- Asking well informed questions.
+
+
+## Identifying Potential Projects
+
+Your goal is to identify potential projects for improving the core Ethereum protocol.  You should spend enough time with each project idea to gain a high level understanding of it.  If you already **know** what you want to work on, you may choose to skip this stage, however, there is value in gaining a broad view of the Ethereum protocol.
+
+## Deliverable
+
+The deliverable will come in two parts.
+
+### 1. Documenting Project Ideas
+
+First you will be hunting for possible projects. You should take a lot of notes.  When you find a potential project, start writing things down and aggregating information.  You can organize this information however you choose, but you should try to make it publicly consumable.  Each project idea will be different, but in general, you should probably try to gather at minimum:
+
+- A "name" for the project
+    - e.g. "Removing Gas Observability aka Ungas", "State Expiry", "Meta Transactions"
+- Prior work: Links to existing forum posts or other documents that people have produced.
+- A high level description of the project/topic/subject.
+    - Even if someone else has already done this, try to do so again using your own words as this is a good way to gauge whether you understand the topic.
+- Anything else that feels relevant.
+
+If you haven't already, fork this repository.  You are free to organize your work however you wish, as long as you do so in a public way.  I'll recomend that you create a document under `./notes/<your-name>.md` where you can take notes and commit.
```

Comment:
Perhaps there should be a `./notes` and `./projects/<user>/<project-name>.md`, to allow a more browseable directory structure, in addition to the master list below?

Reply:
Lets see how this shakes out.  I'm hesitant to add too much structure up front and we can keep this in our back pocket once people actually start populating documents with project information.

Reply:
Makes sense.